integrated_channels/sap_success_factors/client.py

```diff
@@ -31,6 +32,8 @@ class SAPSuccessFactorsAPIClient(IntegratedChannelApiClient):  # pylint: disable
     """
 
     SESSION_TIMEOUT = 5
+    MAX_RETRIES = 3
+    BACKOFF_FACTOR = 1
```

Comment:
can we pick it from configs?
Ref:
https://github.com/openedx/edx-enterprise/blob/8f1d7e18af169e2acd2b625427b2bd74aa42256a/integrated_channels/degreed2/client.py#L40

Reply:
🫡  On it!

Reply:
@hamzawaleed01 I've updated the code please have a look